changelog/dev-fix-account-session

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Fix for validation issue with POST params in some cases generating account session.
```

Comment:
I think it would be better to have just one changelog. What do you think?

Reply:
Is this going to be merged back to `develop`? I assume it won't be because the changes are already in the `develop` branch.